shell/models/storage.k8s.io.storageclass.js

```diff
@@ -78,6 +78,11 @@ export const PROVISIONER_OPTIONS = [
     value:         'driver.harvesterhci.io',
     supported:     true,
     hideCustomize: true,
+  },
+  {
+    labelKey:  'storageClass.outscale-bsu.title',
+    value:     'bsu.csi.outscale.com',
+    supported: true,

```

Comment:
sorry, additional comment, supported will need to be removed as per https://www.suse.com/support/kb/doc/?id=000020481

Unsupported storage classes can be made visible by enabling the `unsupported-storage-drivers` Feature Flag (burger menu --> Global Setting --> Feautre Flags)